test/integration/admin/list/clinicians-all.js

```diff
@@ -2,8 +2,13 @@ import _ from 'underscore';
 
 context('clinicians list', function() {
   specify('display clinicians list', function() {
+    // routeClinicians gets called twice in this app because of the clinicians bootstrap call.
```

Comment:
I think this was the cause of the 404 issue I was having in cypress tests. There may be a better way to handle this, but starting out on another view then navigating to the clinicians list was a reliable workaround. Added this comment anywhere I was doing that for context.